oop/zad-1/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Pierwsza klasa</title>
+  	<meta charset="UTF-8">
```

Comment:
popraw wciecia w html i css, pomimo, ze wydaje sie ok, to u mnie w innym edytorze sie rozjezdza poniewaz sa spacje wymieszane z tabami. Jak zaznaczysz w sublime to widac kropki/kreski: https://scr.hu/8qLMMR

Zeby to ujednolicic wybierz w sublime `View -> Indentation -> Convert indentation to ...`